test/fixtures/es-module-loaders/loader-load-null-return.mjs

```diff
@@ -0,0 +1,3 @@
+export async function load(specifier, context, next) {
```

Comment:
Can we also have a test for when the return value is `null` instead of `Promise<null>`?
```suggestion
export function load(specifier, context, next) {
```

Reply:
Yep! Will do tomorrow (alternatively in the tidy follow-up I'm about to do)

PS gaah, yah killin me with the drip-feeding 😜